python/doc/examples/probabilistic_modeling/stochastic_processes/plot_kronecker_covmodel.py

```diff
@@ -0,0 +1,71 @@
+"""
+Create a kronecker covariance model
+===================================
+"""
+# %%
+# This use case illustrates how the User can create a Kronocker covariance function, which allows defining a multivariate covariance function with a specific covariance structure.
+#  
+# For that purpose, a spatial covariance matrix :math:`\underline{\underline{C}}^{spat}` is needed in addition to a scalar correlation function :math:`\rho`.
+#
+#  We recall that model is defined by
+# .. math: :
+#
+#    C(\vect{s}, \vect{t}) = \rho\left(\dfrac{\vect{s}}{\theta}, \dfrac{\vect{t}}{\theta}\right) C ^ {stat}(\vect{s}, \vect{t}),
+#
+# where the output covariance matrix: math: `C ^ {stat}(\vect{s}, \vect{t})` is given by
+#
+# .. math: :
+#    C ^ {stat}(\vect{s}, \vect{t}) = \mbox{Diag}(\vect{\sigma}) \, \mat{R} \,  \mbox{Diag}(\vect{\sigma}).
+# 
+#   
+# We call :math:`\underline{a}` the amplitude vector and :math:`\underline{\theta}` the scale vector.
+#
+# The library implements the multivariate exponential model thanks to the object *KroneckerCovarianceModel* which is created from:
+#
+#
+# - the correlation function and amplitude vectors :math:`(\underline{\rho}, \underline{a})`: in that case, by default :math:`\underline{\underline{R}} = \underline{\underline{I}}`;
+# - the correlation function and amplitude vectors and the spatial correlation matrix  :math:`(\underline{\rho}, \underline{a},\underline{\underline{R}})`;
+# - the correlation function and amplitude vectors and the spatial covariance matrix  :math:`(\underline{\rho}, \underline{a}, \underline{\underline{C}})`; Then :math:`\underline{\underline{C}}` is mapped into the associated correlation matrix :math:`\underline{\underline{R}}` and the previous constructor is used.
+#
+# Note also that :math:`\underline{\theta}` is taken into account via :math:`(\underline{\rho}``
+#
+
+# %%
+from __future__ import print_function
+import openturns as ot
+import openturns.viewer as viewer
+from matplotlib import pylab as plt
+import math as m
+ot.Log.Show(ot.Log.NONE)
+
+# %%
+# Create the amplitude vector (output dimension 2)
+amplitude = [1.0, 2.0]
+
+# Scale vector (input dimension 1)
+scale = [4.0]
+
+# Create the rho function
+rho = ot.MaternModel(scale, 1.5)
+
+# spatialCorrelation
+spatialCorrelation = ot.CorrelationMatrix(2)
+spatialCorrelation[0, 1] = 0.01
+
+# spatialCovariance
+spatialCovariance = ot.CovarianceMatrix(2)
+spatialCovariance[0, 0] = 4.0
+spatialCovariance[1, 1] = 5.0
+spatialCovariance[1, 0] = 0.5
+
+# %%
+# Create the covariance model from the amplitude and correlation, no spatial correlation
+ot.KroneckerCovarianceModel(rho, amplitude)
+
+# %%
+# or from the correlation function, amplitude, and spatialCovariance
+ot.KroneckerCovarianceModel(rho, amplitude, spatialCorrelation)
+
+# %%
+# or from the correlation model and spatialCovariance
+ot.KroneckerCovarianceModel(rho, spatialCovariance)
```

Comment:
You could perhaps sample a realization of a Gaussian process R --> R^2 with a Kronecker covariance kernel. The marginals could be plotted as 2 time series on the same graph.

Reply:
I used a copy of an example ([this one](https://openturns.github.io/openturns/latest/auto_probabilistic_modeling/stochastic_processes/plot_create_stationary_covmodel.html?highlight=exponentialmodel)). 

If we want to be complete, we might have several examples but not sure to have the time for